src/Events/SignedIn.php

```diff
@@ -37,4 +37,26 @@ public function __construct(Saml2User $user, Auth $auth)
         $this->user = $user;
         $this->auth = $auth;
     }
+
+    /**
+     * Get the authentication handler for a SAML sign in attempt
+     *
+     * @return Auth The authentication handler for the SignedIn event
+     */
+    public function getAuth(): Auth
+    {
+        return $this->auth;
+    }
+
+    /**
+     * Get the user represented in the SAML sign in attempt
+     *
+     * @return Saml2User The user for the SignedIn event
+     */
+    public function getSaml2User(): Saml2User
+    {
+        $this->user = $user;
+        $this->auth = $auth;
```

Comment:
Why is it here? Typo?

Reply:
Yep, sure is! 😞 
Just noticed that after looking at the PR quality report.